### PR TITLE
Added graphics::ElapsedQuery for GPU-profiling

### PR DESCRIPTION
**Pull request by koalefant**
It was only tested on Linux and WASM and problably won't compile yet on other platforms. 

Is it something that we would like to have?

### CODE REVIEW

**Comment by not-fl3** [comment]
Yes, absolutely. 

I could help with compilation errors or just fix them later

**Comment by koalefant** [comment]
I will get Windows fixed at a later point, and maybe Android, but I don't have an Apple workstation at the moment.

**Comment by koalefant** [comment]
I overlooked one important detail: EXT_disjoint_timer_query does not work on Firefox. So for this to be useful one would need ElapsedTimer::is_supported() :disappointed:

**Comment by koalefant** [comment]
Added Windows support.

File: native/sapp-windows/src/gl.rs [hunk 1]
```diff
@@ -598,5 +602,11 @@ gl_loader!(
     fn glGenVertexArrays(n: GLsizei, arrays: *mut GLuint) -> (),
     fn glFrontFace(mode: GLenum) -> (),
     fn glCullFace(mode: GLenum) -> (),
-    fn glGenTextures(n: GLsizei, textures: *mut GLuint) -> ()
+    fn glGenTextures(n: GLsizei, textures: *mut GLuint) -> (),
```
Comment (not-fl3) [inline native/sapp-windows/src/gl.rs:605]: The new gl generator is so much better compared to the nonsense we had before, isn't it? :D

Comment (koalefant) [reply native/sapp-windows/src/gl.rs:605]: Yes, it was a pleasure to do this change with the new loader.

**Comment by not-fl3** [comment]
> I overlooked one important detail: EXT_disjoint_timer_query does not work on Firefox. So for this to be useful one would need ElapsedTimer::is_supported() disappointed

Found it just when I started profiling weird firefox only FPS drop :D  

But fortunately is not actually completely broken in firefox: with "webgl.enable-privileged-extensions:true" EXT_disjoint_timer_query still works!

**Comment by koalefant** [comment]
> webgl.enable-privileged-extensions:true
Yay, now I can profile on Firefox as well! 😊 Did not know about this option.

**Pull request merged**